aquadoggo/src/materializer/service.rs

```diff
@@ -115,9 +120,9 @@ pub async fn materializer_service(
         }
     });
 
-    debug!("Materialiser service is ready");
+    debug!("Materializer service is ready");
```

Comment:
Hui, didn't we decide to write it with "s" 🤣

Reply:
Yeh, we did, but I didn't wanna change module names and code in unrelated places in this PR, so for consistency kept the status quo.